test/native/libs/mallocwrapper.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <stdlib.h>
+ #include <string.h>
+
+ __attribute__((visibility("default")))
+ void* malloc(size_t size) {
+     void* ptr = calloc(1, size);
+     memset(ptr, 0xFF, size);
+     return ptr;
+ }
+ 
```

Comment:
There should be no empty lines in the end, but the last line should terminate with EOL character (check other files too).